src/Dfc.CourseDirectory.Web/Views/AddCourse/AddCourseDetails.cshtml

```diff
@@ -268,43 +268,8 @@
             })
         </div>
 
-
-
-        @{
-            var lastAddCoursePage = HttpContextAccessor.HttpContext.Session.GetObject<AddCoursePage>
-                ("LastAddCoursePage");
-            var summaryPageLoadedAtLeastOnce = HttpContextAccessor.HttpContext.Session.GetObject<bool>
-                ("SummaryLoadedAtLeastOnce");
-            switch (lastAddCoursePage)
-            {
-                case AddCoursePage.AddCourse:
-                    if (summaryPageLoadedAtLeastOnce)
-                    {
-                        <button name="submitButton" type="submit" class="govuk-button" id="publish" value="publish">
-                            Summary
-                        </button>
-                                @*<a class="govuk-link" href="#" id="goToSection1Link">Page 1</a>*@
-                    }
-                    else
-                    {
-                        <button name="submitButton" type="submit" class="govuk-button" id="publish" value="publish">
-                            Continue
-                        </button>
-                                @*<a class="govuk-link" href="#" id="goToSection1Link">Page 1</a>*@
-                    }
-                    break;
-                case AddCoursePage.Summary:
-                    <div class="govuk-form-group">
-                        @*id="goToSummary"*@
-                        <button type="submit" class="govuk-button" id="publish" value="publish">
-                            Summary
-                        </button>
-
-                        <a class="govuk-link" href="#" id="goToSection1Link">Page 1</a>
-                    </div>
-                    break;
-            }
-        }
+        <button name="submitButton" type="submit" class="govuk-button" id="publish" value="publish">Continue</button>
+        @*<a class="govuk-link" href="#" id="goToSection1Link">Page 1</a>*@
```

Comment:
I'd delete the commented out code.

Reply:
Somebody left it in there commented for a reason (perhaps?). I'm trying to tread lightly.

Reply:
Nah, dead code, kill it :fire: